lib/cn/csched_sp3.c

```diff
@@ -2677,7 +2699,7 @@ sp3_tree_add(struct csched *handle, struct cn_tree *tree)
 
     mutex_lock(&sp->mon_lock);
     list_add(&spt->spt_tlink, &sp->add_tlist);
-    atomic_inc_rel(&sp->sp_trees_count);
+    atomic_inc_rel(&sp->sp_addprune_count);
```

Comment:
Var name `trees_count` makes more sense to me than `addprune_count`.

Reply:
But it never was a count of trees being managed by csched (we have no such counter), it's the count of trees being added/pruned.  It's used to skip calling trees_add()/tree_prune() if there aren't any trees to add or pruned, mostly so that we don't have to walk the tree list on every iteration of the monitor just to see if a tree needs to be pruned.  Cheaper just to check the counter and avoid all that unnecessary work...

Reply:
My bad.  I thought it was there from the beginning and counted trees managed my csched.